pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -257,16 +255,15 @@ func (l *podNICImpl) PlugPhase2(vmi *v1.VirtualMachineInstance, iface *v1.Interf
 		return err
 	}
 
-	pid := "self"
-
-	isExist, err := bindMechanism.loadCachedInterface(pid, iface.Name)
+	isExist, err := bindMechanism.loadCachedInterface()
 	if err != nil {
 		log.Log.Reason(err).Critical("failed to load cached interface configuration")
 	}
 	if !isExist {
 		log.Log.Reason(err).Critical("cached interface configuration doesn't exist")
 	}
 
+	pid := "self"
```

Comment:
I found it confusing to see this behave differently from the others and still the helper treating the `self` case.
I guess it would have been solved by not passing the pid at all, like we do with the `*cachedInterface`.

I notice the other comment that discusses this affecting the unit-test, I guess if we want to discuss it, we should do it on that thread.

Reply:
Opening the can of worms of the unit test on this PR will be tricky.

I can:
  - use hard-coded `"self"` 
  - use `getPODString(b.launcherPID) ` 

I go along with both.